angr/analyses/cfg/cfg_accurate.py

```diff
@@ -208,6 +208,10 @@ def __init__(self,
             else:
                 raise AngrCFGError('Unsupported type of the `starts` argument.')
 
+        if enable_advanced_backward_slicing or enable_symbolic_back_traversal:
+            l.warning("`advanced backward slicing` and `symbolic back traversal` are deprecated. Nothing will happen now.")
```

Comment:
"Nothing will happen now" is inaccurate. These two options will still work, right?

Reply:
No the old techniques are never called. Lemme put them back